code/modules/jobs/job_types/cyborg.dm

```diff
@@ -11,6 +11,7 @@
 	minimal_player_age = 21
 	exp_requirements = 120
 	exp_type = EXP_TYPE_CREW
+	alt_titles = list("Android", "Robot", "Cyborg")
```

Comment:
Cyborg is the starter title and is now listed as an alt title as well - "Automaton" perhaps?